streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -142,39 +140,39 @@ public void shouldBeAbleToQueryState() throws Exception {
                 StringSerializer.class,
                 new Properties()));
 
-        final KStream<String, String> s1 = builder.stream(STREAM_ONE);
+        final KStream<String, String> s1 = this.builder.stream(STREAM_ONE);
```

Comment:
`KafkaProducer` will use `this.time = new SystemTime()` to set the timestamp if the `ProducerRecord`'s timestamp is null, will that be okay?